catt/controllers.py

```diff
@@ -94,13 +94,13 @@ def get_stream(url, device_info=None, host=None, ytdl_options=None):
     return StreamInfo(url, host=host, model=cc_info, device_type=cast_type, ytdl_options=ytdl_options)
 
 
-def get_app_info(id_or_name, cast_type=None, strict=False, show_warning=False):
+def get_app(id_or_name, cast_type=None, strict=False, show_warning=False):
     if id_or_name == "default":
         return DEFAULT_APP
 
     arg_is_id = True if re.match("[0-9A-F]{8}$", id_or_name) else False
     try:
-        app_info = next(a for a in APP_INFO_LIST if (a.id if arg_is_id else a.name) == id_or_name)
+        app = next(a for a in APPS if (a.id if arg_is_id else a.name) == id_or_name)
```

Comment:
The generator/`next` thing here might be a bit too confusing to save space on a list of two elements, but that's what we had before, so it's fine.

Reply:
I actually had an if clause with a generator for each argument type initially. Would you prefer that?

Reply:
We're just trying to find the app with the given id, right? Why not just a for/else loop with a return?

Reply:
Or with the given name. I brewed those together to make calls to the function cleaner.

Reply:
Ah okay, I'm not fussy about it. I would have done something like:

```
for app in APPS:
    if id_or_name in (app.id, app.name):
        return app
else:
    return DEFAULT_APP  # todo strict handling etc
```

Which I think looks a bit simpler.

Reply:
I don't like "for loop else". I think it looks weird.

Reply:
You can just not use the else. It'll return if it matches in the loop.